content/docs/setup/kubernetes/mesh-expansion/index.md

```diff
@@ -5,272 +5,280 @@ weight: 50
+7.  Verify the node agent works:
 
     {{< text bash >}}
-    $ curl 'http://10.60.1.4:8080/v1/registration/istio-pilot.istio-system.svc.cluster.local|http-discovery'
-    {{< /text >}}
 
-*   Extract the initial Istio authentication secrets and copy them to the machine. The default
-installation of Istio includes Citadel and will generate Istio secrets even if
-the automatic `mTLS`
-setting is disabled (it creates a secret for each service account, and the secret
-is named as `istio.<serviceaccount>`). It is recommended that you perform this
-step to make it easy to enable mutual TLS in the future and to upgrade to a future version
-that will have mutual TLS enabled by default.
+    $ sudo node_agent
+    ....
+    CSR is approved successfully. Will renew cert in 1079h59m59.84568493s
 
-    `ACCOUNT` defaults to 'default', or `SERVICE_ACCOUNT` environment variable
-    `NAMESPACE` defaults to current namespace, or `SERVICE_NAMESPACE` environment variable
-    (this step is done by machineSetup)
-    On a Mac either `brew install base64` or `set BASE64_DECODE="/usr/bin/base64 -D"`
+    {{< /text >}}
+
+8.  Start Istio using `systemctl`.
 
     {{< text bash >}}
-    $ @install/tools/setupMeshEx.sh@ machineCerts ACCOUNT NAMESPACE
+
+    $ sudo systemctl start istio-auth-node-agent
+    $ sudo systemctl start istio
+
     {{< /text >}}
 
-    The generated files (`key.pem`, `root-cert.pem`, `cert-chain.pem`) must be copied to /etc/certs on each machine, readable by istio-proxy.
+After setup, the machine can access services running in the Kubernetes cluster
+or other mesh expansion machines.
 
-*   Install Istio Debian files and start 'istio' and 'istio-auth-node-agent' services.
-Get the debian packages from [GitHub releases](https://github.com/istio/istio/releases) or:
+The following example shows how to access the services running in the cluster using `/etc/hosts`:
 
-    {{< text bash >}}
-    $ source istio.VERSION # defines version and URLs env var
-    $ curl -L ${PILOT_DEBIAN_URL}/istio-sidecar.deb > istio-sidecar.deb
-    $ dpkg -i istio-sidecar.deb
-    $ systemctl start istio
-    $ systemctl start istio-auth-node-agent
-    {{< /text >}}
+{{< text bash >}}
 
------- Manual setup steps end ------
+$ # On the kubeadm machine
+$ kubectl -n bookinfo get svc productpage -o jsonpath='{.spec.clusterIP}'
+10.55.246.247
 
-After setup, the machine should be able to access services running in the Kubernetes cluster
-or other mesh expansion machines.
+{{< /text >}}
 
 {{< text bash >}}
+
+$ # On the VM
+$ sudo echo "10.55.246.247 productpage.bookinfo.svc.cluster.local" >> /etc/hosts
 $ curl productpage.bookinfo.svc.cluster.local:9080
 ... html content ...
+
 {{< /text >}}
 
-Check that the processes are running:
+## Running services on a mesh expansion machine
+
+VM services are added to the mesh by configuring a
+[ServiceEntry](/docs/reference/config/istio.networking.v1alpha3/#ServiceEntry). A ServiceEntry lets you manually add
```

Comment:
Please note that we are using `ServiceEntry` only to reference a specific instance of a service entry or the `ServiceEntry` kind itself to avoid confusion. When talking about service entries generally, we use normal capitalization:
 ```suggestion
[service entry](/docs/reference/config/istio.networking.v1alpha3/#ServiceEntry). A service entry lets you manually add
```

Reply:
Though aren't we using a ServiceEntry (the configuration object) to add these VM services?